app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/ShowParam.kt

```diff
@@ -0,0 +1,16 @@
+package com.github.palFinderTeam.palfinder.meetups.activities
+
+import com.github.palFinderTeam.palfinder.utils.Gender
+
+/**
+ * simple enum class used to pass parameters to the meetup fetch and retrieve only those matching conditions
+ */
+enum class ShowParam(val param: String) {
+    ALL("all"),
+    ONLY_JOINED("joined"),
+    PAL_PARTCIPATING("pals are participating"),
+    PAL_CREATOR("pal has created it");

```

Comment:
What do you mean by "pal" ?

Reply:
I named Pal the user you are following, so it was shorter to write